### DISCUSSION TITLE
nodeip-configuration.service can't start. http: server gave HTTP response to HTTPS client

### DISCUSSION THREAD

**Discussion by vinisman** [okd-project/okd#1139] [General]
Hi all! 
After configuring my cluster 4.9.0-0.okd-2022-02-12-140851 for private registry i have an error for starting nodeip-configuration.service.

```
level=warning msg="failed, retrying in 1s ... (2/3). Error: initializing source docker://quay.io/openshift/okd-content@sha256:f19112d58ddca7b801e14bbeeca4dc708bab27922680a23a70dda2c9a2db5bb8: (Mirrors also failed: [nexus.mycompany.ru:60001/okd-mirror/okd@sha256:f19112d58ddca7b801e14bbeeca4dc708
bab27922680a23a70dda2c9a2db5bb8: pinging container registry nexus.mycompany.ru:60001: Get \"https://nexus.mycompany.ru:60001/v2/\": http: server gave HTTP response to HTTPS client]):
```
I understand an error and found the bug https://github.com/containers/podman/pull/12064 which is fixed in podman version >= [v3.4.3](https://github.com/containers/podman/releases/tag/v3.4.3)
Can i update the podman version manualy may be or should I wait for okd automatic update only? Also I can't find any version or patch for okd where this should be fixed.

**Comment by vrutkovs** [comment]
All host content is controlled by OKD's machine-os-content image. You may make changes via enabling Fedora repos and running `rpm-ostree install/update` - but this might break future upgrades

**Comment by vinisman** [comment]
> 
Ok, so I have a choice of
1) Manual package update for node
2) Configure https protocol for my registry (prefered)

Do you plan to add this bug to any update?

**Reply by vrutkovs** [reply]
Podman updates would be picked up from Fedora during regular okd-machine-os updates.
